docs/design/abstractions/project-structure.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: "Project Structure"
+description: "Details about how the Mage file directory works and how typical projects are structured."
+---
+
+## Overview
+
+A Mage project lives inside it's own folder. It's there where mage data is stored (`mage_data`) and individual projects are housed. You can think of a project as an environment: it contains configurations, pipelines, and other project-specific data. A [workspace](production/developing-in-the-cloud/workspaces/overview) can hold multiple projects.
```

Comment:
1. The link with relative path doesn't work.
2. A workspace is for one project. That doc is about managing multiple workspaces.